kuma/settings/content_experiments.json

```diff
@@ -1,2 +1,85 @@
 [
+    {
+        "id": "experiment-interactive-editor",
+        "ga_name": "interactive-editor",
+        "param": "v",
+        "pages": [
```

Comment:
See test_views_document for the current format of this setting:

https://github.com/mozilla/kuma/blob/fc03aad8490991553c149633c2f0280693b6b9ec/kuma/wiki/tests/test_views_document.py#L18-L29

``pages`` is now a dict of ``locale:slug`` keys to a dict of ``variant``, ``slug`` pairs.